src/test/java/org/sonarlint/intellij/AbstractSonarLintLightTests.java

```diff
@@ -60,8 +62,11 @@ protected final String getTestDataPath() {
     return Paths.get("src/test/testData/" + this.getClass().getSimpleName()).toAbsolutePath().toString();
   }
 
-  @Before
-  public final void init() {
+  @BeforeEach
+  final void beforeEachLightTest() throws Exception {
+    // dummy test name, not used
+    setName("test");
```

Comment:
What's the purpose of this?

Reply:
IntelliJ test framework is built around JUnit 3 & 4, to make it work with JUnit 5 I have to explicitly call `TestCase.setName(String)`.
Thanks for your comment, I changed this to provide the real test name instead of dummy, it will be cleaner.